src/shared/utils/countries.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Provides lookup data for countryies.
+ */
+import countries from './countries.json';
```

Comment:
For such functionality we should rely on some standard external package, and not to implement it ourselves.